cpu/stm32/include/periph_cpu.h

```diff
@@ -83,6 +83,11 @@ extern "C" {
  */
 #define TIMER_CHAN          (4U)
 
+/**
+ * @brief   Define a macro for accessing a timer channel
+ */
+#define TIM_CHAN(tim, chan) *(uint32_t *)(&dev(tim)->CCR1 + chan)
```

Comment:
```suggestion
#define TIM_CHAN(tim, chan) *(&dev(tim)->CCR1 + chan)
```
You can drop the cast - just squash directly if you decide so.

Reply:
Done